data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/DroidKaigi2021Api.kt

```diff
@@ -433,33 +439,61 @@ internal fun SessionAllResponse.toTimetableContents(): TimetableContents {
                 TimetableSpeaker(apiSpeaker.fullName!!, apiSpeaker.profilePicture)
             }.first()
         }
+    val categoryIdToCategory: Map<Int, TimetableCategory> = feedContents.categories!!
+        .flatMap { it.items!! }
```

Comment:
It is also possible to reference only "Category" in categories, in which case we can use the order or id of "Category". 
The current implementation targets the items contained in all categories.

Reply:
Note: In the current JSON, "sessionCategoryItemId" is 89435, which specifies the "Session format".

Reply:
I checked too. I think it's okay. 🙆‍♀️